### PR TITLE
!!! TASK: Load Neos Routes from settings

### PR DESCRIPTION
**Pull request by bwaidelich**
This adjusts the Configuration to include Neos default routes
via Settings.

Background:
Since version 3.1 Flow supports sub routes to be included via configuration
(see http://flowframework.readthedocs.io/en/stable/TheDefinitiveGuide/PartIII/Routing.html#subroutes-from-settings).
This is the recommended way to register routes as it provides greater flexibility.
Note: Any existing `Routes.yaml` file in the global `Configuration` folder take
precedence over routes loaded via Settings.

This change should not break existing installations, but it might lead to routes being loaded twice
or (in rare cases) in a different order.
To make sure, any existing `Routes.yaml` file should be removed from the global `Configuration`
folders and the respective routes should be loaded via Settings (see documentation above).

In order to change the "defaultUriSuffix" from Neos Routes, add the following to your
`Settings.yaml`:

    Neos:
      Flow:
        mvc:
          routes:
            'Neos.Neos':
              variables:
                'defaultUriSuffix': ''

(an empty string will remove the suffix)

Resolves: #1342
Related: neos/flow-development-collection#789

### CODE REVIEW

File: Neos.Neos/Configuration/Settings.yaml [hunk 1]
```diff
@@ -560,6 +560,11 @@ Neos:
       applicationPackageKey: Neos.Neos
       applicationName: Neos
 
+    mvc:
+      routes:
+        'Neos.Neos':
+          position: 'start'
```
Comment (aertmann) [inline Neos.Neos/Configuration/Settings.yaml]: how about the defaultUriSuffix variable?

**Comment by aertmann** [comment]
#1305 takes care of removing media browser routes as well

**Comment by aertmann** [comment]
need to get rid of https://github.com/neos/neos-development-distribution/blob/master/Configuration/Routes.yaml and https://github.com/neos/neos-base-distribution/blob/master/Configuration/Routes.yaml as well

File: Neos.Neos/Configuration/Settings.yaml (hunk 1 above)
Comment (johannessteu) [inline Neos.Neos/Configuration/Settings.yaml]: And `position: 'start'` seems to be breaking, custom routes, wich should always be loaded before neos subroutes should still be loaded before neos.neos

File: Neos.Neos/Configuration/Settings.yaml [hunk 2]
```diff
@@ -560,6 +560,10 @@ Neos:
       applicationPackageKey: Neos.Neos
       applicationName: Neos
 
+    mvc:
+      routes:
+        'Neos.Neos': TRUE
```
Comment (aertmann) [inline Neos.Neos/Configuration/Settings.yaml]: how about `defaultUriSuffix`?

Comment (bwaidelich) [inline Neos.Neos/Configuration/Settings.yaml]: Good catch!

**Pull request merged**